e2e-tests/insurance/cypress/e2e/journeys/export-contract/change-your-answers/agent-charges/change-your-answers-agent-charges-fixed-sum-amount.spec.js

```diff
@@ -0,0 +1,79 @@
+import { field, summaryList } from '../../../../../../../pages/shared';
+import { INSURANCE_ROUTES } from '../../../../../../../constants/routes/insurance';
+import FIELD_IDS from '../../../../../../../constants/field-ids/insurance/export-contract';
+import application from '../../../../../../../fixtures/application';
+
+const {
+  ROOT,
+  EXPORT_CONTRACT: {
+    CHECK_YOUR_ANSWERS,
+    AGENT_CHARGES_CHANGE,
+  },
+} = INSURANCE_ROUTES;
+
+const {
+  AGENT_CHARGES: { FIXED_SUM_AMOUNT },
+} = FIELD_IDS;
+
+const fieldId = FIXED_SUM_AMOUNT;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context(`Insurance - Export contract - Change your answers - Agent charges - ${FIXED_SUM_AMOUNT} As an Exporter, I want to be able to review my input regarding the amount an agent is charging for helping me win my export contract, So that I can be assured I am providing UKEF with the right information`, () => {
```

Comment:
not a blocker, missing - after FIXED_SUM_AMOUNT

Reply:
Good spot, will fix in the next PR